test/verify/check-multi-machine-key

```diff
@@ -230,6 +230,7 @@ class TestMultiMachineKeyAuth(MachineCase):
                                     ".*: bridge program failed: Child process exited with code .*",
                                     # Since there is not password,
                                     # reauthorize doesn't work on m2
+                                    ".*: received authorize command for wrong user: user",
```

Comment:
The test still [failed](https://fedorapeople.org/groups/cockpit/logs/pull-6179-a7484c26-verify-debian-testing/log.html#97) -- this message doesn't start with anything `.*:` (i. e. no colon):

    Unexpected journal message 'received authorize command for wrong user: user'